### ISSUE TITLE
Unable to slow down during Moon landing

### ISSUE THREAD

**Issue by itsArk55** [GTNewHorizons/GT-New-Horizons-Modpack#16667]
### Your GTNH Discord Username

@marka999

### Your Pack Version

2.6.1

### Your Server

Private server

### Java Version

Java 21

### Type of Server

Don't know

### Your Expectation

To slow down during moon landing as normal with holding spacebar

### The Reality

https://youtu.be/V9CI5rxFMsM?si=B4NtNWaH30_JDaBJ

feel free to skip to 1:30

### Your Proposal

Fix or remove lander for moon

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
did you maybe change some keybinds?

**Comment by itsArk55** [comment]
No, definitely havent changed the spacebar keybind and the landing works sometimes but others happens as in video.Sent from my iPhoneOn 2 Jul 2024, at 20:47, chochem ***@***.***> wrote:﻿
did you maybe change some keybinds?

**Comment by Officialy** [comment]
It happened to me just now, was holding space and didn't slow down, pressed it numerously to see if any hold attempts registered. 
It's worked on and off once or twice, but seems unreliable for me.

**Comment by boscook** [comment]
It happened to me just now, too, I have checked the key blind and tested in singleplayer, singleplayer everything is working, but in my private server is not working

**Comment by LynxAmurie** [comment]
I get this issue every time I try to land on the moon on my private server, but only the first time after the server has been started. I've tried with the landing chunks forced to be loaded and it still occurs. Not observed to happen in SSP. I haven't changed any keybinds relevant to player locomotion, and there is no explicit keybind for GC's landing sequence.

**Comment by PLASMAchicken** [comment]
> I get this issue every time I try to land on the moon on my private server, but only the first time after the server has been started. I've tried with the landing chunks forced to be loaded and it still occurs. Not observed to happen in SSP. I haven't changed any keybinds relevant to player locomotion, and there is no explicit keybind for GC's landing sequence.

GTNH Version? Java Versin? Launcher?

**Comment by BritishCynic** [comment]
This happens to me, too, but only in multiplayer. Have tested it using Prism on 2.8.4 (Server on J8, Client on J25), and confirmed that it happens when connected to my MP world, but not when in SP.

**Comment by TheDarkDnKTv** [comment]
Having the same issue on a private server (ping ~120ms). Java 8, GTNH 2.8.4, vanilla launcher

**Comment by PLASMAchicken** [comment]
I mean just down use Java 8 ....
